pom.xml

```diff
@@ -62,9 +62,9 @@
         <maven.assembly.plugin.version>2.4</maven.assembly.plugin.version>
         <maven.rar.plugin.version>2.2</maven.rar.plugin.version>
         <maven.bundle.plugin.version>2.4.0</maven.bundle.plugin.version>
-        <maven.shade.plugin.version>3.0.0</maven.shade.plugin.version>
+        <maven.shade.plugin.version>3.2.4</maven.shade.plugin.version>
         <maven.dependency.plugin.version>2.10</maven.dependency.plugin.version>
-        <maven.animal.sniffer.plugin.version>1.15</maven.animal.sniffer.plugin.version>
+        <maven.animal.sniffer.plugin.version>1.19</maven.animal.sniffer.plugin.version>
```

Comment:
Without upgrading the animal-sniffer plugin, it failed with a cryptic `InvalidArgumentException` with no reference to what exactly failed. Afterwards, the shade plugin failed so I upgraded it, which caused the change in the method signature for `HazelcastManifestTransformer`